.circleci/config.yml

```diff
@@ -165,7 +165,7 @@ commands:
             command: |
               cd ..
               git clone git@github.com:facebookresearch/Mephisto.git Mephisto
-              cd Mephisto; git checkout v0.3.1.1 -b stable
+              cd Mephisto; git checkout 2e8e131a8c27ccabeccc4af894777d1b2ca449fe -b stable
```

Comment:
i still don't like these.

we should be pip installing here, not checking out code

Reply:
The hardcoded commit SHA here will be changed to a clean tag name before merging in the PR (I left a note in the PR description about this). But I'd definitely support `pip install`ing a specific version of Mephisto - @JackUrb do you know if that is supported?

Reply:
We're close to the pip install, just a few rough edges I haven't figured out yet. Should be there by the end of the month.